ModelBundle/Document/Site.php

```diff
@@ -318,4 +318,23 @@ public function getMainAliasId()
     {
         return $this->aliases->indexOf($this->getMainAlias());
     }
+
+    /**
+     * Return alias id for given language
+     *
+     * @param string $language
+     *
+     * @return int
+     */
+    public function getAliasIdForLanguage($language)
```

Comment:
As we said in the planning, we should (for the moment) get only the first alias with this language.

This method is temporary and we will improve it in the following sprints I think